internal/app/handlers/post_handler.go

```diff
@@ -0,0 +1,29 @@
+package handlers
+
+import (
+	"mishin-shortener/internal/app/hasher"
+	"mishin-shortener/internal/app/storage"
+	"io"
+	"net/http"
+)
+
+func PostHandler(w http.ResponseWriter, r *http.Request, db *storage.Database, redirectHost string) {
+	if r.RequestURI == "/" {
+
+		body, err := io.ReadAll(r.Body)
+		if err != nil {
+			panic(err)
```

Comment:
Не стоит использовать panic(...) в хендлере, так как это приведет к краху приложения. Я рекомендую обработать ошибку, залогировать ее и вернуть сообщение об ошибке.
```suggestion
			http.Error(w, "Internal Server Error", http.StatusInternalServerError)
			return
```

Reply:
Согласен, убрал